pkg/go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/getoutreach/stencil-golang/pkg
+
```

Comment:
We're using a sub module here to prevent all importers from having to import stencil